rllib/core/columns.py

```diff
@@ -43,7 +43,9 @@ class Columns:
 
     # Common extra RLModule output keys.
     STATE_IN = "state_in"
+    NEXT_STATE_IN = "new_state_in"
```

Comment:
Let's use "next" everywhere.

We should also change the "new_obs" at some point into "next_obs", but not in this PR.

Reply:
Alrighty. I change here the `new_state_in` into `next_state_in`.